test/test_rpc_manager.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2018 Palantir Technologies, Inc.
+import pytest
+import time
+from StringIO import StringIO
```

Comment:
`io.StringIO`

Reply:
StringIO isn't the right way, but neither is io.StringIO. it forces you to have to convert all your strings before sending them off, which is totally unnecessary during normal execution

Reply:
This will not run on Python 3 and this server is compatible is both Python 2 and 3. You need to:
```python
import sys

if sys.version_info[0] < 3:
    from StringIO import StringIO

else:
    from io import StringIO
```